src/testsuite/vec_pwr9_dummy.c

```diff
@@ -1576,7 +1586,7 @@ example_longdiv_10e31_PWR9 (vui128_t *q, vui128_t *d, long int _N)
 
   return rh;
 }
-
+#ifndef __clang__
```

Comment:
why? add a comment.

Reply:
Actually it the several things but:
// Clang does not support vec_bcd_ppc.h